test_runner/scenario_test_runner/package.xml

```diff
@@ -18,7 +18,7 @@
   <exec_depend>openscenario_utility</exec_depend>
 
   <!-- AutowareAuto launch -->
-  <!-- <exec_depend>scenario_test_runner_launch</exec_depend> -->
```

Comment:
Please let me know where is the scenario_test_runner package is?

Reply:
It is [here](https://gitlab.com/robotecai-public/AutowareAuto/-/merge_requests/9), to be merged with `scenario_simulator_v2_changes` branch